keyboards/xd75/keymaps/buzzlighter1/readme.md

```diff
@@ -1,41 +1,40 @@
 # XD75 with 7U spacebar for EN-RU gamers
 
-Standard QWERTY oriented on gamers with a Russian alternative input.
-Base layout is as close to classic layout as possible due to RU alphabet having more letters (thus the need for those extra keys remain on the right side).
-Full uncut 2x4 nav cluster.
-Caps Lock and Win keys sent to the Function layer.
-TT(1) - Function layer tap-toggle at CapsLock position provides fast F-keys access in a heat of a battle. As well as easy access to a Numpad.
+Standard QWERTY made for gamers with a Russian alternative input.
+The keys for extra letters in RU alphabet are where they are expected.
+Full 2x4 nav cluster. 
+Volume/mute control are on base layer, Win key is on function layer.
+FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
+NKRO is working (forced).
 
-/* QWERTY
```

Comment:
Rendering is kinda bad here still, you might want to revert and wrap with triple backticks,

    ```c
    /* QWERTY
     * .--------------------------------------------------------------------------------------------------------------------------------------.
     * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
    ...
     */
    ```

will produce

```c
/* QWERTY
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
...
 */
```

Reply:
Still outstanding.

Reply:
# XD75 with 7U spacebar for EN-RU gamers
Standard QWERTY made for gamers with a Russian alternative input.
The keys for extra letters in RU alphabet are where they are expected.
Full 2x4 nav cluster. 
Volume/mute control are on base layer, Win key is on function layer.
FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
NKRO is working (forced).
## Layout
### Qwerty layout
```
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
 * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | [      | ]      | \      | Mute   |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
 * | FN(CPS)| A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | '      | Vol-   | Vol+   | ENTER  |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
 * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
 * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
 * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```
### Function (FN)
```
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | XXXXXXX| F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | F11    | F12    | _______| _______|
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P7     | P8     | P9     | +      | P*     | XXXXXXX| PR SCR | SCR LK | PAUSE  | XXXXXXX| BL_DEC | BL_INC | RGB HD | RGB HI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
```
## LEDs ID
Top left LED - Function layer active.
Mid left LED - CapsLock active.

Reply:
nope, see <https://github.com/qmk/qmk_firmware/blob/db4b11f866477791c15de75a72d6444bdb0e6449/keyboards/xd75/keymaps/buzzlighter1/readme.md>

Reply:
Do you have an example with proper formatting?

Reply:
<https://github.com/zvecr/qmk_firmware/blob/1bc30a161fd41b96052588e702f88fe1b134cf44/keyboards/xd75/keymaps/buzzlighter1/readme.md>

Reply:
# XD75 with 7U spacebar for EN-RU gamers
 * Standard QWERTY made for gamers with a Russian alternative input.
 * The keys for extra letters in RU alphabet are where they are expected.
 * Full 2x4 nav cluster.
 * Volume/mute control are on base layer, Win key is on function layer.
 * FN_CAPS provides fast F-keys access while gaming. Lefthanded Numpad operation.
 * NKRO is on (forced).

## Layout
```c
/* QWERTY
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | GESC   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      | -      | =      | Del    | BACKSP |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
 * | TAB    | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | [      | ]      | \      | Mute   |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
 * | FN(CPS)| A      | S      | D      | F      | G      | H      | J      | K      | L      | ;      | '      | Vol-   | Vol+   | ENTER  |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
 * | LSHIFT | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Home   | PGUP   | UP     | PGDWN  |
 * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
 * | LCTRL  | Del    | ENTER  | LALT   |                             SPACE                            | End    | LEFT   | DOWN   | RIGHT  |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
 */
```
```c
/* FUNCTION
 * .--------------------------------------------------------------------------------------------------------------------------------------.
 * | XXXXXXX| F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | F11    | F12    | _______| _______|
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P7     | P8     | P9     | +      | P*     | XXXXXXX| PR SCR | SCR LK | PAUSE  | XXXXXXX| BL_DEC | BL_INC | RGB HD | RGB HI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P4     | P5     | P6     | NumLk  | CALC   | XXXXXXX| POWER  | XXXXXXX| RESET  | XXXXXXX| BL_BRTG| BL_STEP| RGB SD | RGB SI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P1     | P2     | P3     | -      | P/     | XXXXXXX| OS     |  PLAY  | PREV   | NXT    | BL_OFF | BL_ON  | RGB VD | RGB VI |
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
 * | _______| P0     | P.     | PENT   |                             SPACE                            | BL_TOGG| RGB TG | RGB RMD| RGB MD |
 * '--------------------------------------------------------------------------------------------------------------------------------------'
 */
```
## LEDs
Top left LED - Function layer active.
Mid left LED - CapsLock active.

## Compile

go to qmk top directory.
```
$ cd qmk_firmware
```

build
```
$ make xd75:buzzlighter1
```

Reply:
Yeah thats what it looks like fixed with my proposed changes.

Your current instance does not render correctly on github within firefox or chrome, windows and linux, can we just make the changes and move on please.

Reply:
I'm sorry, is the latest version OK or not? What should I do now?
Previously I was just accepting edits. Now I can only see and option to comment (that is why I have sent the file as a comment).